app/containers/Button/index.tsx

```diff
@@ -9,7 +9,7 @@ import ActivityIndicator from '../ActivityIndicator';
 interface IButtonProps extends PlatformTouchableProps {
 	title: string;
 	onPress: () => void;
-	type?: string; // primary | secondary
+	type?: 'primary' | 'secondary'; // primary | secondary
```

Comment:
If we go this way, let's remove the comment, while fixing the need for `as const` as well.